staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher.go

```diff
@@ -434,6 +439,13 @@ func NewCacherFromConfig(config Config) (*Cacher, error) {
 		)
 	}()
 
+	if utilfeature.DefaultFeatureGate.Enabled(features.ConsistentListFromCache) {
+		supports := cacher.Supports(storage.RequestWatchProgress)
+		if !supports {
+			klog.InfoS("RequestWatchProgress feature is not supported by provided etcd. This will prevent ConsistentListFromCache from working")
```

Comment:
```suggestion
			klog.InfoS("RequestWatchProgress feature is not supported by provided etcd. This will prevent ConsistentListFromCache from working", "resource", cacher.groupResource.String())
```

Reply:
Let's add which resource this cacher is responsible for